### PR TITLE
Refactored testcases

### PR DESCRIPTION
**Pull request by mmazb3**

### CODE REVIEW

**Comment by mmazb3** [comment]
I tested each test case in debug mode and confirmed they worked. 
Need to figure out how to solve the add_user_button not pressing on time

**Comment by mmazb3** [comment]
Test cases now work without debug mode. Made the following changes:
1. Add_user_button is clicked only after the list of all users is displayed. Added if statement to click_add_user() to check this condition.
2. Created input_password() in step.py which is based on input_text() but sends the keys twice. Otherwise, password strength check messages didn't appear.

File: test/test_case_6_Verify_Password_Field_Required_Validation.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+# Purpose: To verify that the password strength indicator shows 'Strongest' when the password includes a mix of uppercase letters, lowercase letters, numbers, and special symbols.
+# Steps:
+# 1. Follow steps 1-4 from Test Case 6.
+# 2. Enter 'Aa1!Aa1!' in the 'Password' field.
+# 3. Observe the password strength indicator.
+# Expected Result:
+# The password strength indicator should display a 'Strongest' message.
+
+
+
```
Comment (bersh92) [inline test/test_case_6_Verify_Password_Field_Required_Validation.py:70]: as you moved all tests here fill free to delete separate files

File: test/test_case_6_1.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def test_case_6_1_Verify_Password_Field_Required_Validation(app):
```
Comment (bersh92) [inline test/test_case_6_1.py:1]: you missed the name of the .py file

File: test/test_case_6_Verify_Password_Field_Required_Validation.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+# Test Case 6_6: Verify Password Strength Indicator - Strongest
+# Test Name: Test_Password_Strength_Strongest
+# Purpose: To verify that the password strength indicator shows 'Strongest' when the password includes a mix of uppercase letters, lowercase letters, numbers, and special symbols.
+# Steps:
+# 1. Follow steps 1-4 from Test Case 6.
+# 2. Enter 'Aa1!Aa1!' in the 'Password' field.
+# 3. Observe the password strength indicator.
+# Expected Result:
+# The password strength indicator should display a 'Strongest' message.
+
```
Comment (bersh92) [inline test/test_case_6_Verify_Password_Field_Required_Validation.py:68]: please run all tests together with script and verify that everything is fine. Then after resolve all comments you can merge the pull request

File: test/test_case_6_1.py (hunk 2 above)
Comment (mmazb3) [reply test/test_case_6_1.py:1]: Oh, I am sorry, this is a temporary file with a single test case I used for testing. Please disregard it.

**Pull request merged**